filters/quantitative_ques/filter.py

```diff
@@ -0,0 +1,33 @@
+import spacy
+from interfaces.QuestionAnswerOperation import QuestionAnswerOperation
+from tasks.TaskTypes import TaskType
+
+
+class QuantitativeQuestion(QuestionAnswerOperation):
+    tasks = [TaskType.QUESTION_ANSWERING, TaskType.QUESTION_GENERATION]
+    languages = ["en"]
+
+    def __init__(self):
+        super().__init__()
+        self.nlp = spacy.load("en_core_web_sm")
+        # Covers the broad types of quant questions: distance , age , measurable , un-measurable
+        self.quant_ques = ['many','much',
+                           'close','far',
+                           'young','old',
+                           'short','tall',
+                           'heavy','light',
+                           'narrow','wide',
+                           'deep','shallow',
+                           'broad','thin',
+                           'near','long']
+
+
+    def filter(self,context:str = None,question: str = None,answers:str = None) -> bool:
+        tokenized = self.nlp(question, disable=["parser", "tagger", "ner"])
+        if 'how' == tokenized[0].text.lower() and tokenized[1].text.lower() in self.quant_ques:
```

Comment:
Minor code formatting: Instead of multiline, we could change it to a single line check.

return ('how' == tokenized[0].text.lower() and tokenized[1].text.lower() in quant_ques)

Reply:
Thanks for the review @aadesh11 

I have made the changes, could you please review it again

Reply:
Thanks, it's fine now.